### PR TITLE
Renamed invalid parameter in docstring

### PR DESCRIPTION
**Pull request by nikssardana**
Renamed result_list to result in the docstring of print_affected_files

Closes https://github.com/coala/coala/issues/2809

### CODE REVIEW

**Comment by sils1297** [comment]
ack https://github.com/coala/coala/pull/2820/commits/b0918fd0dba35966b4b717d97a913c74475fff12

File: coalib/output/ConsoleInteraction.py [hunk 1]
```diff
@@ -321,7 +321,7 @@ def print_affected_files(console_printer,
     :param console_printer: Object to print messages on the console.
     :param log_printer:     Printer responsible for logging the messages.
     :param section:         The section to which the results belong to.
-    :param result_list:     List containing the results
+    :param result:          List containing the results
```
Comment (sils1297) [inline coalib/output/ConsoleInteraction.py]: it's not a list of results anymore! Content of the comment needs changing as well

Comment (nikssardana) [inline coalib/output/ConsoleInteraction.py]: What should be the content now? Should it be something like:
param result:          The actual result

**Comment by nikssardana** [comment]
@jayvdb I closed this pull request and made a new one https://github.com/coala/coala/pull/2824. I was having some problems with squashing the commits. Please review that one!

**Pull request closed without merging**